### DISCUSSION TITLE
歡迎來到 work_crawler 討論區！ Welcome to work_crawler Discussions!

### DISCUSSION THREAD

**Discussion by kanasimi** [kanasimi/work_crawler#577] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 👋 歡迎!
  我們使用 GitHub Discussions 作為社區成員聯系的地方。我們希望您：
  * 提出想解決的問題。
  * 分享想法或建議。
  * 與其他社區成員交流。
  * 歡迎他人，並保持開放的心態。請記住，這是我們共同建立的社區💪。

  如需開始，在下方用評論介紹您自己，並告訴我們您對什麼感興趣或者想要做哪些有趣的事。

**Comment by cclvkk** [comment]
在最新的.exe中下载网站飞扑网的URL有问题

**Reply by kanasimi** [reply]
大概是網站改版了。
